sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -509,6 +526,11 @@ object ScalaNativePluginInternal {
     files.filterNot(f => isOtherGC(f.getPath().toString, nativelib))
   }
 
+  private val isWindows: Boolean = {
+    val os = Option(sys props "os.name").getOrElse("")
```

Comment:
`sys.props` will be deprecated in Scala 2.13. Use `System.getProperties` instead. Also, the dot for qualified access is preferred.